README.md

```diff
@@ -24,68 +24,69 @@ For instance, if a module performs HTTP requests to a CouchDB server or makes HT
+    + [Including Content-Length Header Automatically](#including-content-length-header-automatically)
+    + [Including Date Header Automatically](#including-date-header-automatically)
+  * [HTTP Verbs](#http-verbs)
+  * [Support for HTTP and HTTPS](#support-for-http-and-https)
+  * [Non-standard ports](#non-standard-ports)
+  * [Repeat response n times](#repeat-response-n-times)
+  * [Delay the response body](#delay-the-response-body)
+  * [Delay the response](#delay-the-response)
+  * [Delay the connection](#delay-the-connection)
+  * [Socket timeout](#socket-timeout)
+  * [Chaining](#chaining)
+  * [Scope filtering](#scope-filtering)
+  * [Path filtering](#path-filtering)
+  * [Request Body filtering](#request-body-filtering)
+  * [Request Headers Matching](#request-headers-matching)
+  * [Optional Requests](#optional-requests)
+  * [Allow **unmocked** requests on a mocked hostname](#allow-unmocked-requests-on-a-mocked-hostname)
 - [Expectations](#expectations)
-  - [.isDone()](#isdone)
-  - [.cleanAll()](#cleanall)
-  - [.persist()](#persist)
-  - [.pendingMocks()](#pendingmocks)
-  - [.activeMocks()](#activemocks)
-  - [.isActive()](#isactive)
+  * [.isDone()](#isdone)
+  * [.cleanAll()](#cleanall)
+  * [.persist()](#persist)
+  * [.pendingMocks()](#pendingmocks)
+  * [.activeMocks()](#activemocks)
+  * [.isActive()](#isactive)
 - [Logging](#logging)
 - [Restoring](#restoring)
 - [Activating](#activating)
 - [Turning Nock Off (experimental!)](#turning-nock-off-experimental)
 - [Enable/Disable real HTTP requests](#enabledisable-real-http-requests)
-  - [Disabling requests](#disabling-requests)
-  - [Enabling requests](#enabling-requests)
-  - [Resetting NetConnect](#resetting-netconnect)
+  * [Disabling requests](#disabling-requests)
+  * [Enabling requests](#enabling-requests)
+  * [Resetting NetConnect](#resetting-netconnect)
 - [Recording](#recording)
-  - [`dont_print` option](#dont_print-option)
-  - [`output_objects` option](#output_objects-option)
-  - [`enable_reqheaders_recording` option](#enable_reqheaders_recording-option)
-  - [`logging` option](#logging-option)
-  - [`use_separator` option](#use_separator-option)
-  - [.removeInterceptor()](#removeinterceptor)
+  * [`dont_print` option](#dont_print-option)
+  * [`output_objects` option](#output_objects-option)
+  * [`enable_reqheaders_recording` option](#enable_reqheaders_recording-option)
+  * [`logging` option](#logging-option)
+  * [`use_separator` option](#use_separator-option)
+  * [.removeInterceptor()](#removeinterceptor)
 - [Events](#events)
-  - [Global no match event](#global-no-match-event)
+  * [Global no match event](#global-no-match-event)
 - [Nock Back](#nock-back)
-  - [Setup](#setup)
-    - [Options](#options)
-  - [Usage](#usage-1)
-    - [Options](#options-1)
-    - [Modes](#modes)
+  * [Setup](#setup)
+    + [Options](#options)
+  * [Usage](#usage-1)
+    + [Options](#options-1)
+      - [Example](#example)
+    + [Modes](#modes)
```

Comment:
@paulmelnikow Will these bullets work with prettier? If not we can configure `toc-me` with a `.github/toc.yml` file: https://github.com/pholleran/toc-me#configuration-options

Reply:
I think Prettier wants them all with dashes. It's cool it's configurable, though!

Reply:
okay I'll take care of it!

Related question: shouldn’t we run a prettier check as part of the `lint` stage on Travis? I expected it to fail, but it only runs the eslint checks

Reply:
Yea, that's a good idea. It runs as part of the main tests right now.

Also I wonder if we could run lint after tests, or in parallel with, instead of before. I appreciate seeing the lint errors separately though it would be nice if my tests ran too, and anything we can do to shorten the iteration cycle is good.

Reply:
I usually run lint tests before to reduce the time the travis build take, but if you prefer to change it please go ahead, I don’t have strong feelings about it :)

Reply:
How does running it before reduce the time?

Reply:
when it fails, it doesn’t continue the running the other builds